### PR TITLE
operator functionality and db table display 

### PR DESCRIPTION
**Pull request by shankajarajapakse**
*abiout.php added
*database values displayed in tables

### CODE REVIEW

File: Operator/booking.php [hunk 1]
```diff
@@ -1,16 +1,86 @@
-
+<?php
+	$username = "Siripala"
```
Comment (pushpithaDilhan) [inline Operator/booking.php:2]: is this the details of user logged in ? comment when using these

File: Operator/validatebooking.php [hunk 2]
```diff
@@ -1,29 +1,26 @@
...
 	$bus_id=$_POST["bus_id"];
 	$seats =$_POST["seat_count"];
 	$date  = date('Y-m-d H:i:s');
 	$start =$_POST["start"];
 	$end   =$_POST["ending"];
+	$data_complete = 0;
+
 
 	$query ="INSERT INTO booking ( customer_id , bus_id ,seat_range , start , ending ,travel_date ) VALUES ( '{$cus_id}','{$bus_id}','{$seats}','{$start}','{$end}','{$date}' )";
+	$query1 ="INSERT INTO customer ( customer_id ,name ) VALUES ( '{$cus_id}','{$name}')";
```
Comment (pushpithaDilhan) [inline Operator/validatebooking.php:18]: customer kiyla table ekak danatamath hadala tye. mekata wena namak use karanna. e table eke ai aai store karanne.

File: Operator/booking.php (hunk 1 above)
Comment (shankajarajapakse) [reply Operator/booking.php:2]: yeah 
need to update it with a post method

File: Operator/validatebooking.php (hunk 2 above)
Comment (shankajarajapakse) [reply Operator/validatebooking.php:18]: Can you give me ur customer table functionality
???

File: Operator/booking.php (hunk 1 above)
Comment (pushpithaDilhan) [reply Operator/booking.php:2]: use
session_start();
$_SESSION['role_id']

File: Operator/validatebooking.php (hunk 2 above)
Comment (pushpithaDilhan) [reply Operator/validatebooking.php:18]: create table customer
	(name		varchar(100), 
	 NIC			varchar(10), 
	 address		varchar(200),
     occupation     varchar(20),
     email          varchar(100),
     username       varchar(32),                // md5 hashed
     password       varchar(32),
     primary key (username),                // md5 hashed
	 
	);

Comment (pushpithaDilhan) [reply Operator/validatebooking.php:18]: mn meken karanne cutomer details save karala tya gannawa. tawa table ekk tye login kiyla ekatath username / password add weno. eken aran balala tamai login eka wenne

**Pull request merged**